server/moodle/mod/livequiz/tests/phpunit/livequiz_service_test.php

```diff
@@ -497,7 +497,7 @@ public function test_delete_question_throws_if_relation_exist(): void {
      * @throws Exception
      */
     public function delete_answer(): void {
```

Comment:
This is not run, since it is does not contain the prefix test_

In our current branch the test fails, please fix